queries/clean_categories.sql

```diff
@@ -0,0 +1,104 @@
+-- Copyright 2020 Google LLC
```

Comment:
Let's discuss this query a bit, unless you think you know what you want to do based on the comment on the config file.

I also think this could be clearer using WITH, but don't worry about that until handling the comment in the config file.

Reply:
I have hard-coded the category cleaning in this query. But we can discuss using WITH to make this cleaner.